Source/WTF/Scripts/Preferences/WebPreferencesExperimental.yaml

```diff
@@ -1337,9 +1337,9 @@ ReportingEnabled:
     WebKitLegacy:
       default: false
```

Comment:
Don't we support WebKitLegacy? For CSP for e.g., does it only work for WK2?

Reply:
Yes, I guess we do. We just don't generally add new functionality to WebKitLegacy as it is deprecated, and we don't want to encourage adoption.

Reply:
I have been told that we do enable new features on WebKitLegacy as long as they work without too much extra work.

Reply:
I see. I'll go ahead and add it for WebKitLegacy while landing, then.